### ISSUE TITLE
[4.6.4] Hibernate warnings(HHH90000022) about deprecated usages 

### ISSUE THREAD

**Issue by tuxmaster5000** [obiba/opal#3813]
### This issue is unique

- [X] I have used the [search tool](https://github.com/obiba/Opal/issues?q=) and did not find an issue describing my bug.

### Version information

4.6.4

### Expected behavior

Not warnings about deprecated calls.

### Actual behavior

Lot of warnings in the log file:
2023-10-18 08:41:39,165 [qtp217062446-5485] WARN  org.hibernate.orm.deprecation - HHH90000022: Hibernate's legacy org.hibernate.Criteria API is deprecated; use the JPA javax.persistence.criteria.CriteriaQuery instead

### Reproduction steps

There is no "standard way" to reproduce it. It will shown randomly in the log.

### Operating System (OS)

Linux

### Browser

FireFox

**Comment by ymarcon** [comment]
We do not recommend using the Hibernate based SQL schema, use "Tabular SQL" schema instead. Hibernate is hard to maintain and this SQL schema has poor performance.

**Comment by tuxmaster5000** [comment]
Ok, that can be, but how to convert the existing "Opal SQL" to "Tabular SQL" schema? I don't see any convert function.

**Comment by ymarcon** [comment]
There is no "conversion" per say, you can use the project backup/restore feature instead. Or copy the tables into another project.

**Comment by tuxmaster5000** [comment]
I will forward this to the scientists for testing.

**Comment by sofiasiamp** [comment]
Hi, and thank you for your help!
I tried to do the "conversion": 
**with project backup/restore:**
- Created a new folder in the file system called "backup"
- Backed up the data from the project that used OpalSQL into the folder "backup". 
- Opened the backup folder and inside there are two folders: "files" with the data dictionary and dataset and "tables" with a data.csv and a table.json file, meaning that the back up was successful.
- I created a new project (using Tabular SQL) and tried to use the restore function with the backup folder I had created, but both folders files and tables look empty, even though they are not. Therefore, I can't proceed with the restore.

**with copy:**
When attempting to copy a table from one project to another, it works. I can see all values and variables. The only issue is that I get an error in the Tasks tab: 
![image](https://github.com/obiba/opal/assets/104575409/2871790c-9ddc-47a4-8b68-1887204c1bce)
